next-docs/public/examples/snackbarTW/Queue.tsx

```diff
@@ -0,0 +1,90 @@
+import { Snackbar, Button } from '@heathmont/moon-core-tw';
+import { useState, useCallback } from 'react';
```

Comment:
Naming of the file should probably be Snackbar.tsx, not Queue.tsx

Reply:
It's already under `snackbar` directory so it's ok (from my perspective)